doc/md/guides/orms/gorm.md

```diff
@@ -290,7 +290,66 @@ Observe a new migration file is generated:
 -- Modify "pets" table
 ALTER TABLE `pets` ADD COLUMN `nickname` longtext NULL;
 ```
+### View
 
+A database view is a virtual table based on the result of a query. Views are helpful when you want to simplify complex queries, strengthen security by selecting only the necessary data, or encapsulate the details of your table structures.
+
+> Making liberal use of views is a key aspect of good SQL database design. Views allow you to encapsulate the details of your table structures, which might change as your application evolves, behind consistent interfaces.
+> 
+> — <cite>**Postgres documentation**</cite>
```

Comment:
This is great for a blog post. Less so in "dry" tech docs. Keep it in your "stash" for your upcoming post :-)